src/symbols/Label.tsx

```diff
@@ -30,6 +30,36 @@ export interface LabelProps {
    */
   stroke?: ColorRepresentation;
 
+  /**
+   * Background color of the label.
+   */
+  backgroundColor?: ColorRepresentation;
+
+  /**
+   * Opacity of the background.
+   */
+  backgroundOpacity?: number;
+
+  /**
+   * Padding around the text for background sizing.
+   */
+  backgroundPadding?: number;
```

Comment:
I feel like just saying `padding` would be a better name. Background padding is kinda funky.